grapher/core/Grapher.tsx

```diff
@@ -1446,6 +1448,18 @@ export class Grapher
         return this.stackMode === StackMode.relative
     }
 
+    @computed get isFacetYAxisShared(): boolean {
+        return this.facetYScale === FacetAxisRange.shared
+    }
+
+    get facetYScale(): FacetAxisRange {
+        return this.yAxis.facetAxisRange
+    }
+
+    set facetYScale(mode: FacetAxisRange) {
+        this.yAxis.facetAxisRange = mode
+    }
```

Comment:
Seems like the getter is only used in the computed, and the setter not at all. Do we really need them? The original property is only ~6 characters more to access. 

As a developer the getter/setter also make me think I should be using this interface to control the axis. Is that a good idea? Isn't controlling directly through `.yAxis` better?

Reply:
Yeah, agreed also. Cleaned up now.